modules/userId/index.js

```diff
@@ -287,7 +285,7 @@ function storedConsentDataMatchesConsentData(storedConsentData, consentData) {
   return (
     typeof storedConsentData === 'undefined' ||
     storedConsentData === null ||
-    utils.deepEqual(storedConsentData, makeStoredConsentDataObject(consentData))
+    storedConsentData == makeStoredConsentDataHash(consentData)
```

Comment:
Triple equals is probably safer?

Reply:
yea, looking into that.

Reply:
thanks for the suggestion...i've implemented this now